#!/usr/bin/env python
PACKAGE = "bobcat_model"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
# used in wheeled bobcat.
gen.add("joint_control_P",	double_t,	0,	"Porptional",	100000,	0,  10000000)
gen.add("joint_control_I",	double_t,	0,	"Porptional",	0.0,    0,  10000000)
gen.add("joint_control_D",	double_t,	0,	"Porptional",	10000,	0,  10000000)
gen.add("hydraulics_control_P",	double_t,	0,	"Porptional",	10000000,	0,  10000000)
gen.add("hydraulics_control_I",	double_t,	0,	"Porptional",	100,    0,  10000000)
gen.add("hydraulics_control_D",	double_t,	0,	"Porptional",	1000000,	0,  10000000)
gen.add("HydraulicSpeed",	double_t,	0,	"Porptional",	0.1,	0,  2)
gen.add("JointSpeed",	double_t,	0,	"Porptional",	0.25,	0,  2)

exit(gen.generate(PACKAGE, "bobcat_arm_reconfig_node", "bobcat_arm"))
